api/agentpb/agent.proto

```diff
@@ -167,6 +167,13 @@ message StartActionRequest {
   // PTSummaryParams describes parameters for PT summary.
   message PTSummaryParams {}
 
+  // PTMgDbSummaryParams describes parameters for PT MongoDB summary.
+  message PTMgDbSummaryParams {
+    string address = 1;
```

Comment:
it should be renamed to host.

Reply:
please do the same in other PRs.

Reply:
All of you have approved the "address" field in previous two PRs. There was a discussion at the beginning on these fields' names, it was also changed from "addr" to "address".
I will change it in this PR. The others were approved, therefore in my opinion a correct action would be to file a ticket (two) to change it. What do you think?

Reply:
We discussed it with Marcos and agreed that it should be done as part of these PRs, because we won't be able to change it in later releases because of backward compatibility and there is high possibility that we won't deliver your new changes in the same release with current PRs.

Reply:
I'm on it.